src/components/tabs/tabs.scss

```diff
@@ -167,6 +167,11 @@ md-pagination-wrapper {
     width: auto;
     margin: 0 auto;
   }
+  &.md-align-end {
+    position: relative;
+    width: auto;
+    margin: 0 0 0 auto; 
```

Comment:
Yes @EladBezalel , it would. Also `direction` does not change the default alignment of the tabs, only the order is affected. So there wouldn't be any conflict in `rtl`.